tests/integration/cairo1/out.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
`out.json` isn't super descriptive.

Reply:
Will rename to `compiled_test_suite.json`